.evergreen/benchmarks.yml

```diff
@@ -150,7 +147,20 @@ functions:
           export MONGODB_URI="${MONGODB_URI}"
           export SSL="${SSL}"
           . .evergreen/generate-uri.sh
+    - command: expansions.update
+      params:
+        file: src/uri-expansions.yml
 
+  "run driver benchmarks":
+    - command: shell.exec
```

Comment:
can this be subprocess.exec instead?

Reply:
Diff to the contrary, this is just preserved from the existing stanza; switching to subprocess.exec will require sorting out `${PREPARE_SHELL}` (it does more than just re-exporting variables) and I think that'd be better done for all the commands in this file as its own chunk of work.